addons/website/static/src/less/website.snippets.less

```diff
@@ -53,9 +53,32 @@ table.table_desc tr td {
 }
 
 // Other snippet styles
+.s_feature_grid p{
+    overflow: hidden;
+}
+
 .s_comparisons .panel .list-group-item:not(.active) {
     color: #666;
 }
+.s_comparisons {
+    .panel small {
+        color:inherit;
```

Comment:
Because, by default small tag is [grey](https://github.com/odoo/odoo/blob/master/addons/web/static/lib/bootstrap/css/bootstrap.css#L1218) colored in bootstrap, which causes this issue : 
![compare-small-color](https://cloud.githubusercontent.com/assets/7599908/26618260/58e629c6-45f7-11e7-8f66-bba77ce7744d.png)